src/Utils/AreaUtils.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+namespace AppBundle\Utils;
+
+use AppBundle\Intl\FranceCitiesBundle;
+
+class AreaUtils
+{
+    const CODE_CORSICA_A = '2A';
+    const CODE_CORSICA_B = '2B';
+    const CODE_FRANCE = 'FR';
+    const CODE_MONACO = '06';
+    const POSTALCODE_MONACO = '98000';
+    const PREFIX_POSTALCODE_CORSICA = '20';
+    const PREFIX_POSTALCODE_CORSICA_A = ['200', '201'];
+    const PREFIX_POSTALCODE_DOM = '97';
+    const PREFIX_POSTALCODE_PARIS_DISTRICTS = '75';
+    const PREFIX_POSTALCODE_TOM = '98';
+
+    public static function getCodeFromPostalCode(string $postalCode): string
+    {
+        $department = substr($postalCode, 0, 2);
+
+        switch ($department) {
+            case self::PREFIX_POSTALCODE_PARIS_DISTRICTS:
+                return $postalCode;
+            case self::PREFIX_POSTALCODE_TOM:
+                if (self::POSTALCODE_MONACO === $postalCode) {
+                    return self::CODE_MONACO;
+                }
+
+                return substr($postalCode, 0, 3);
+            case self::PREFIX_POSTALCODE_DOM:
+                return substr($postalCode, 0, 3);
+            case self::PREFIX_POSTALCODE_CORSICA:
+                if (in_array(substr($postalCode, 0, 3), self::PREFIX_POSTALCODE_CORSICA_A)) {
+                    return self::CODE_CORSICA_A;
+                }
+
+                return self::CODE_CORSICA_B;
+            default:
+                return $department;
+        }
+    }
+
+    public static function getCodeFromCountry(string $country): string
+    {
+        if (!in_array($country, FranceCitiesBundle::$countries)) {
+            return $country;
+        }
+
+        $code = (string) array_search($country, FranceCitiesBundle::$countries);
```

Comment:
Can you please add `\` for array and string functions everywhere?

Reply:
Why not ? it is necessary to prefix all native array / string functions in the project ?

Reply:
Micro optimization I think

Reply:
Because if you don't use the `\` namespace on function call, PHP is gonna try to look for it in the current NameSpace (just like Classes)

Reply:
Be careful not to do it with `time` function otherwise the time mocker of phpunit bridge would not work.

Reply:
I think is a very very small performance optimization. But if it's really needed, we can add the `\` everywhere in the project in another PR.

Reply:
It's not that small, when you consider the number of calls ;). Please keep it in mind for new code, until a huge dedicated PR.

Reply:
Ok, done for this class :+1: